ONSdigital/response-operations-ui
# [TBL] Upload CI direct to service

# Motivation and Context

Removes the call to backstage for uploading collection instruments.

# What has changed



Changes the `upload_collection_instrument` method in collection_instrument_controllers.py to hit the collection instrument service directly.

# How to test?



Run the unit tests, make sure they pass.
Tail the logs for backstage, collection instrument service. Go to a collection exercise's detail page and upload a new (spreadsheet type) collection instrument. Make sure backstage doesn't get the call, but collection instrument service does.
